security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisParameters.java

```diff
@@ -61,12 +76,132 @@ public static SecurityAnalysisParameters load(PlatformConfig platformConfig) {
         return parameters;
     }
 
+    protected static void load(SecurityAnalysisParameters parameters) {
+        load(parameters, PlatformConfig.defaultConfig());
+    }
+
+    protected static void load(SecurityAnalysisParameters parameters, PlatformConfig platformConfig) {
+        Objects.requireNonNull(parameters);
+        Objects.requireNonNull(platformConfig);
+
+        platformConfig.getOptionalModuleConfig("security-analysis-default-parameters")
+                .ifPresent(config -> {
+                    parameters.setIncreasedFlowViolationsThreshold(config.getDoubleProperty("increasedFlowViolationsThreshold", DEFAULT_INCREASED_FLOW_VIOLATIONS_THRESHOLD));
+                    parameters.setIncreasedLowVoltageViolationsThreshold(config.getDoubleProperty("increasedLowVoltageViolationsThreshold", DEFAULT_INCREASED_LOW_VOLTAGE_VIOLATIONS_THRESHOLD));
+                    parameters.setIncreasedHighVoltageViolationsThreshold(config.getDoubleProperty("increasedHighVoltageViolationsThreshold", DEFAULT_INCREASED_HIGH_VOLTAGE_VIOLATIONS_THRESHOLD));
+                    parameters.setIncreasedLowVoltageViolationsDelta(config.getDoubleProperty("increasedLowVoltageViolationsDelta", DEFAULT_INCREASED_LOW_VOLTAGE_VIOLATIONS_DELTA));
+                    parameters.setIncreasedHighVoltageViolationsDelta(config.getDoubleProperty("increasedHighVoltageViolationsDelta", DEFAULT_INCREASED_HIGH_VOLTAGE_VIOLATIONS_DELTA));
+                });
+    }
+
+    public SecurityAnalysisParameters(double increasedFlowViolationsThreshold, double increasedLowVoltageViolationsThreshold, double increasedHighVoltageViolationsThreshold,
+                                      double increasedLowVoltageViolationsDelta, double increasedHighVoltageViolationsDelta) {
+        this.increasedFlowViolationsThreshold = increasedFlowViolationsThreshold;
+        this.increasedLowVoltageViolationsThreshold = increasedLowVoltageViolationsThreshold;
+        this.increasedHighVoltageViolationsThreshold = increasedHighVoltageViolationsThreshold;
+        this.increasedLowVoltageViolationsDelta = increasedLowVoltageViolationsDelta;
+        this.increasedHighVoltageViolationsDelta = increasedHighVoltageViolationsDelta;
+    }
+
+    public SecurityAnalysisParameters() {
+        this(DEFAULT_INCREASED_FLOW_VIOLATIONS_THRESHOLD, DEFAULT_INCREASED_LOW_VOLTAGE_VIOLATIONS_THRESHOLD, DEFAULT_INCREASED_HIGH_VOLTAGE_VIOLATIONS_THRESHOLD,
```

Comment:
No need to define default values here AND in the field definitions.

Reply:
Done